manifests/d/Dev47Apps/DroidCamOBS/1.5.1/Dev47Apps.DroidCamOBS.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Dev47Apps.DroidCamOBS
+PackageVersion: 1.5.1
+PackageLocale: en-US
+Publisher: Dev47Apps

```

```suggestion
Publisher: Dev47apps
```
This must match Control Panel (including the capitalization differences)